From 1479ab4f8a794677353d0f63cb72706b5a58ea63 Mon Sep 17 00:00:00 2001
From: Etienne Carriere <etienne.carriere@foss.st.com>
Date: Thu, 4 Jul 2024 18:51:22 +0200
Subject: [PATCH 1170/1695] dts: stm32: stm32mp257f-ev1 ommanager does not load
 pinctrl

Remove pinctrl configuration from OM Manager device node since the
OM Manager driver in OP-TEE does not need a pinctrl which pins are
all not assigned to OP-TEE, preventing them to be used by OP-TEE.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 8c1c5622a47e99abbf76bc3ae82bb2118bfd00bb dts: stm32: stm32mp257f-ev1 ommanager does not load pinctrl]
Change-Id: Id982a246c8d4b75927704cff5b5668dfa114aee8
Signed-off-by: Etienne Carriere <etienne.carriere@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/395470
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
Domain-Review: Christophe KERELLO <christophe.kerello@st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 core/arch/arm/dts/stm32mp257f-ev1.dts | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/core/arch/arm/dts/stm32mp257f-ev1.dts b/core/arch/arm/dts/stm32mp257f-ev1.dts
index 3aec9a8ec..dbcafe138 100644
--- a/core/arch/arm/dts/stm32mp257f-ev1.dts
+++ b/core/arch/arm/dts/stm32mp257f-ev1.dts
@@ -349,13 +349,6 @@
 &ommanager {
 	memory-region = <&mm_ospi1>;
 	memory-region-names = "mm_ospi1";
-	pinctrl-names = "default", "sleep";
-	pinctrl-0 = <&ospi_port1_clk_pins_a
-		     &ospi_port1_io03_pins_a
-		     &ospi_port1_cs0_pins_a>;
-	pinctrl-1 = <&ospi_port1_clk_sleep_pins_a
-		     &ospi_port1_io03_sleep_pins_a
-		     &ospi_port1_cs0_sleep_pins_a>;
 	status = "okay";
 };
 
-- 
2.47.2

